website/docs/cloud-docs/users-teams-organizations/users.mdx

```diff
@@ -120,8 +120,6 @@ Any of these actions requires you to re-authenticate regardless of session timeo
 
 Click **Organizations** in the sidebar to view a list of the organizations where you are a member. If you are on the [owners team][owners], the organization is marked with an **OWNER** badge.
 
-To leave an organization, click the **ellipses (...)** next to the organization and select **Leave organization**. You do not need permission from the owners to leave an organization, but you cannot leave if you are the last member of the owners team. Either add a new owner and then leave, or [delete the organization](/terraform/cloud-docs/users-teams-organizations/organizations#general).
```

Comment:
Is it intentional that we're not replacing this with the same instructions as above? If we don't think this is something user's will need/expect on this page I'm fine with it, just checking.

Reply:
I debated this one back and forth, but now that I'm giving it a second though I think I might add it back. There's technically two ways to leave an organization now: the User Settings page and the Organizations overview. It would make sense to document both :)